addons/mail/static/tests/qunit_suite_tests/components/channel_member_list_tests.js

```diff
@@ -143,5 +143,66 @@ QUnit.test('there should be a button to hide member list in the thread view topb
     );
 });
 
+QUnit.test('should show a button to load more members if they are not all loaded', async function (assert) {
+    assert.expect(1);
+
+    const pyEnv = await startServer();
+    const channel_last_seen_partner_ids = [[0, 0, { partner_id: pyEnv.currentPartnerId }]];
+    for (let i = 0; i < 101; i++) {
+        const resPartnerId = pyEnv['res.partner'].create({ name: "name" + i });
+        channel_last_seen_partner_ids.push([0, 0, { partner_id: resPartnerId }]);
+    }
+    const mailChannelId = pyEnv['mail.channel'].create({
+        channel_last_seen_partner_ids,
+        channel_type: 'group',
+        public: 'private',
+    });
+    const { click } = await this.start({
+        discuss: {
+            context: {
+                active_id: `mail.channel_${mailChannelId}`,
+            },
+        },
+    });
+    await click('.o_ThreadViewTopbar_showMemberListButton');
+    assert.containsOnce(
+        document.body,
+        '.o_ChannelMemberList_loadMoreButton',
+        "should have a load more button",
+    );
+});
+
+
+QUnit.test('Load more button should load more members', async function (assert) {
+    assert.expect(1);
+
+    const pyEnv = await startServer();
+    const channel_last_seen_partner_ids = [[0, 0, { partner_id: pyEnv.currentPartnerId }]];
+    for (let i = 0; i < 101; i++) {
+        const resPartnerId = pyEnv['res.partner'].create({ name: "name" + i });
+        channel_last_seen_partner_ids.push([0, 0, { partner_id: resPartnerId }]);
+    }
+    const mailChannelId = pyEnv['mail.channel'].create({
+        channel_last_seen_partner_ids,
+        channel_type: 'group',
+        public: 'private',
+    });
+    const { click } = await this.start({
+        discuss: {
+            context: {
+                active_id: `mail.channel_${mailChannelId}`,
+            },
+        },
+    });
+    await click('.o_ThreadViewTopbar_showMemberListButton');
+    await click('.o_ChannelMemberList_loadMoreButton');
+    assert.containsN(
+        document.body,
+        '.o_ChannelMember',
+        102,
+        "should load all the members",
```

Comment:
"because 100 members were fetched initially, and load more fetched the remaining 2 members"